obrobot_navigation/PoseOperation poseoperation
---
string status
obrobot_navigation/PoseDefine posedefine
